.gitignore

```diff
@@ -17,4 +17,4 @@ src/org/olap4j/mdx/parser/impl/Token.java
 src/org/olap4j/mdx/parser/impl/TokenMgrError.java
 testlib/
 eclipse-bin/
-
+pom.xml.bak
```

Comment:
The pom sort plugin creates a backup incase it fudges it, I figure it good
to create the backup, not so good to ingest it.

--------------

Director Meteorite.bi - Saiku Analytics Founder
Tel: +44(0)5603641316

(Thanks to the Saiku community we reached our Kickstart
<http://kickstarter.com/projects/2117053714/saiku-reporting-interactive-report-designer/>
goal, but you can always help by sponsoring the project
<http://www.meteorite.bi/products/saiku/sponsorship>)

On 30 March 2016 at 17:15, Luc Boudreau <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/olap4j/olap4j/pull/42#discussion_r57917312>:
>
> > @@ -17,4 +17,4 @@ src/org/olap4j/mdx/parser/impl/Token.java
> >  src/org/olap4j/mdx/parser/impl/TokenMgrError.java
> >  testlib/
> >  eclipse-bin/
> > -
> > +pom.xml.bak
>
> Probably not needed :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/olap4j/olap4j/pull/42/files/17834b2663d7381ea850503800feb56893f91fc7#r57917312>
>